app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ArticlesController < ApplicationController
+  
+  http_basic_authenticate_with name: "user", password: "secret", except: [:index, :show]
```

Comment:
- What is the purpose of this method `http_basic_authenticate_with`?
- Use double quotes only when you need String Interpolation.
- Use `%i` notation for arrays of symbols.

Reply:
1. The purpose is making a kind of authentication, with the user called "user" and a password. for some resources in this case the restriction is applied to _Create_, _Edit_ and Delete _Articles_.
2. I will change the strings to a single quote.
3. I review the use of %i and I will change it.